docs/conf.py

```diff
@@ -0,0 +1,48 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+"""
+Configuration file for mewbot sphinx document build.
+"""
+
+project = 'mewbot'
+copyright = '2023, Mewler Org'
+author = 'Mewler Org'
+release = '0.0.1'
```

Comment:
To raise an a separate issue: need to work out the source of truth for these values.

Reply:
Ohh - yes. Good point. I'll add it as an issue.

Reply:
Added as issue https://github.com/mewler/mewbot/issues/173